src/axis.ts

```diff
@@ -98,6 +103,12 @@ export interface AxisConfig {
    * @minimum 0
    */
   subdivide?: number;
+
+  /**
+   * Whether the axis should include ticks.
```

Comment:
I think include seems fine here, especially given how many dependent PRs we have.